assets/helpers/user.js

```diff
@@ -1,12 +1,63 @@
 // @flow
```

Comment:
Since this is likely to increment, I think it would be better to split all the parts into different files. Create a User folder and put UserAction and UserReducer.